plugins/board-resources/src/components/move-selectors/SpaceSelect.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+    import { IntlString, translate } from '@anticrm/platform'
```

Comment:
I am not sure, but maybe it is better to create a folder `selectors` and move it there?

Reply:
Done